spec/lexers/make_spec.rb

```diff
@@ -22,4 +22,13 @@
       assert_guess :source => '.PHONY: all'
     end
   end
+
+  describe 'lexing' do
+    include Support::Lexing
+
+    it 'recognizes declarations not terminated by a new line (#694)' do
+      assert_tokens_equal "hello: \n\techo hello",
+       ["Name.Label", "hello"], ["Operator", ":"], ["Text", " \n\t"], ["Name.Builtin", "echo "], ["Text", "hello"]
```

Comment:
It was `[["Name.Label", "hello"], ["Operator", ":"], ["Text", " \n\t"], ["Error", "echo hello"]]` as issued.